schema/mysql/upgrades/002.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE schedule ADD COLUMN timezone text;
```

Comment:
This could additionally specify `AFTER name` to match the column order of the full schema. (PostgreSQL doesn't provide something like that though.)